internal/rpmmd/repository.go

```diff
@@ -128,10 +134,10 @@ func (re *RepositoryError) Error() string {
 func LoadRepositories(confPaths []string, distro string) (map[string][]RepoConfig, error) {
 	var f *os.File
 	var err error
-	path := "/repositories/" + distro + ".json"
+	repoPath := "/repositories/" + distro + ".json"
 
 	for _, confPath := range confPaths {
-		f, err = os.Open(confPath + path)
+		f, err = os.Open(confPath + repoPath)
```

Comment:
Is this an unrelated change?

Reply:
This was because I was using the `path` library but I have now switched to use `path/filepath`